include/stdlib.h

```diff
@@ -62,7 +62,11 @@
  * function call.  However, get_environ_ptr() can be used in its place.
  */
 
-#define environ get_environ_ptr()
+#ifdef CONFIG_DISABLE_ENVIRON
```

Comment:
Logic here is inverted. Should be ifndef

Reply:
Yes. You are right

Reply:
Done, thanks.